libs/api/src/main/java/de/cyface/api/MeasurementRetrievalImpl.java

```diff
@@ -57,8 +57,8 @@ public MetaData metaData(final JsonObject document) {
         final var osVersion = metaData.getString("osVersion");
         final var appVersion = metaData.getString("appVersion");
         final var length = metaData.getDouble("length");
-        final var username = metaData.getString("username");
-        return new MetaData(identifier, deviceType, osVersion, appVersion, length, username, version);
+        final var userId = metaData.getString("userId");
```

Comment:
I wasn't sure if I should use `ObjectId` as `userId` Class or `String`.

First I went with `ObjectId` to get the extra checks of mongo db ObjectId but then decided, that the `User` model should not really depend on the implementation (database id type).

Reply:
makes sense